package.json

```diff
@@ -20,7 +20,8 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^1.7.0",
+    "@theforeman/vendor": "^2.0.0",
+    "connected-react-router": "^6.6.0",
```

Comment:
I think the vendor is more like a shareable dependencies wrapper across plugins, but this one should be used only in core. I'd like to prevent a redundant plugins update when an irrelevant package has a breaking changes update.

Reply:
as discussed offline, added it to vendor in https://github.com/theforeman/foreman-js/pull/82